plutus-conformance/test-cases/uplc/evaluation/builtin/semantics/integerToByteString/little-endian/bounded/maximum-width-zero/maximum-width-zero.uplc

```diff
@@ -0,0 +1,4 @@
+-- Check that we can encode zero using the maximum width (8000).
+(program 1.0.1
+ [(builtin integerToByteString) (con bool False) (con integer 8000) (con integer 0)]
```

Comment:
If you're going to make it 8192, don't forget to update this test. And I guess plenty of other ones. So maybe it's easier to just keep 8000.